File: C#/Trivia/Trivia/Question.cs
namespace Trivia {
    public class Question
    {

        public int id;
        public int category;
        public string question;
        public string answer;
        public int answeredBy;

        public Question(int id, int category, string question, string answer)
        {
            this.id = id;
            this.category = category;
            this.question = question;
            this.answer = answer;
        }
    }
}

File: kotlin/src/main/kotlin/com/adaptionsoft/games/uglytrivia/Question.kt
package com.adaptionsoft.games.uglytrivia

enum class Category {
    POP,
    SCIENCE,
    SPORTS,
    ROCK
}

data class Question(val category: Category, val id: Int) {
    override fun toString(): String = "${toCategory(category)} Question $id"

    private fun toCategory(category: Category): String = category.name.toLowerCase().capitalize()
}
